docs/source/en/features/pipeline_parallel.md

```diff
@@ -3,12 +3,11 @@
 Author: Guangyang Lu, Hongxin Liu, Yongbin Li
 
 **Prerequisite**
-- [Define Your Configuration](../basics/define_your_config.md)
-- [Use Engine and Trainer in Training](../basics/engine_trainer.md)
-- [Configure Parallelization](../basics/configure_parallelization.md)
+- [Use Booster to Training](../basics/booster_api.md)
```

Comment:
Add `../concepts/paradigms_of_parallelism` and `../basics/booster_plugin.md` to prerequisite